packages/about/lib/components/about-view.js

```diff
@@ -160,12 +167,31 @@ module.exports = class AboutView extends EtchComponent {
         )
       ),
 
-      $(UpdateView, {
-        updateManager: this.props.updateManager,
-        availableVersion: this.props.availableVersion,
-        viewUpdateReleaseNotes: this.handleReleaseNotesClick.bind(this),
-        viewUpdateInstructions: this.handleHowToUpdateClick.bind(this)
-      }),
+      $.div(
+        { className: 'about-updates group-start' },
+        $.div(
+          { className: 'about-updates-box' },
+          $.div(
+            { className: 'about-updates-status' },
+            $.div(
+              { className: 'about-updates-item app-unsupported' },
+              $.span(
+                { className: 'about-updates-label is-strong' },
+                'Updates have been moved to the package `pulsar-updater`',
```

Comment:
```suggestion
                'Updates have been moved to the package ', $.code({}, 'pulsar-updater'), '.',
```

Style tweak?
Before:
![Backticks - Updates have been moved to the package `pulsar-updater`](https://github.com/pulsar-edit/pulsar/assets/20157115/ec74cc7d-00e6-4160-aee1-8ed47c6f7dc2)
After:
![Code element - Updates have been moved to the package pulsar-updater](https://github.com/pulsar-edit/pulsar/assets/20157115/d831c829-e53f-448e-a062-d5a10560ab00)


This makes the line ~94 characters long, if you want me to indent it to make the line length shorter I can re-jigger it a bit.

Reply:
Ehh, this works for me, thanks!